Directory.Packages.props

```diff
@@ -113,8 +112,8 @@
               '$(MSBuildProjectFile)' == 'NuGet.VisualStudio.Contracts.csproj' OR
               '$(MSBuildProjectFile)' == 'NuGet.VisualStudio.Interop.csproj' OR
               '$(MSBuildProjectFile)' == 'NuGet.SolutionRestoreManager.Interop.csproj'">
-    <PackageVersion Include="Microsoft.ServiceHub.Framework" Version="2.7.327-preview" />
-    <PackageVersion Include="Microsoft.VisualStudio.ComponentModelHost" Version="17.0.67-g9e37b637e6" />
+    <PackageVersion Include="Microsoft.ServiceHub.Framework" Version="4.1.112" />
```

Comment:
I was actually working on moving this to 4.2 because there's a breaking change in 17.6, but great point about having us depend on packages that are only on nuget.org!

I'll try to account for that in my changes.

Reply:
now I see you have a phase that validates it!

Awesome job!

Reply:
> I was actually working on moving this to 4.2 because there's a breaking change in 17.6, but great point about having us depend on packages that are only on nuget.org!

Using package dependencies on nuget.org is only important for packages we publish to nuget.org. We could theoretically use different package versions in different projects. However, the VS SDK consumption feed's 401 responses makes it difficult in practice :(   we can't (easily/feasibly) have a different nuget.config (package source mapping config) for different projects.

Anyway, point is that if we need to upgrade the VSSDK or Microsoft.ServiceHub.Framework version for our VS extension, we theoretically can. We just have to find a solution to the infrastructure issues.